tests/lib/test_constructor.py

```diff
@@ -235,6 +235,10 @@ def tzname(self, dt):
         def dst(self, dt):
             return datetime.timedelta(0)
 
+    class MyFullLoader(yaml.FullLoader):
+        def get_state_keys_blacklist(self):
+            return super().get_state_keys_blacklist() + ['^mymethod$', '^wrong_.*$']
```

Comment:
@ret2libc I believe this is what is causing CICD to fail. Since this is Python 2, should be
```
return super(MyFullLoader, self)...
```
Right? Please change and see if the tests pass on travis and appveyor at the bottom of the PR page...

Cheers!

Reply:
Ops, sorry about that.